__tests__/unit-tests/getTitle.test.js

```diff
@@ -0,0 +1,23 @@
+const getTitle = require('../../src/utils/getTitle');
+
+describe('Get Title Method', () => {
+  let title = '';
+
+  it('should return the title if there is no alt tag', () => {
+    title = 'Article title';
+    const value = getTitle(title);
+    expect(value).toBe(title);
+  });
+
+  it('should filter the title and return everything inside the alt tag', () => {
+    title = `<img src="//thetimes.co.uk/abracadabra" alt="Article title">`;
+    const value = getTitle(title);
```

Comment:
Looking at this makes me excited to try out some TDD! 🤟

Reply:
Yeah,  that would be awesome ✌️